app/models/health/careplan.rb

```diff
@@ -263,6 +263,14 @@ def compact_future_issues
       end
     end
 
+    def future_issues?
+      (0..10).each do |i|
```

Comment:
Should this be (0..9) like in goal base?

Reply:
Nope... careplan table has has 11 columns for future issues (0-10 inclusive). The goals table has 10 columns for action steps (0-9 inclusive).